ui/src/app/views/profile-page/component.html

```diff
@@ -19,110 +19,109 @@ <h2>Profile</h2>
-          </div>
-          <span class="alert-text">Your identity has been verified.</span>
-        </div>
+    <h5>Username:</h5>
+    <div *ngIf="profileLoaded else loading">{{profile.username}}</div>
+    <ng-template #loading>
+      <div class="loading-area">
+        <span class="loading-text">Loading...</span>
+      </div>
+    </ng-template>
+    <button *ngIf="editing" (click)="submitChanges()" class="btn">Update Profile</button>
+    <button *ngIf="editing" (click)="editing=false" class="btn">Cancel</button>
+  </div>
+  <div *ngIf="profileLoaded" style="width: 13rem">
+    <div class="card">
+      <div style="display: flex; justify-content: space-around">
+        <img [src]="profileImage" style="border-radius: 100px">
+        <h6 style="padding: 0.8rem">
+          {{profile.givenName}} {{profile.familyName}} Profile Image
+        </h6>
       </div>
     </div>
-    <div class="alert alert-danger" *ngIf="profile.blockscoreIdVerificationStatus === 'REJECTED'">
-      <div class="alert-items">
-        <div class="alert-item static">
-          <div class="alert-icon-wrapper">
-            <clr-icon class="alert-icon" shape="exclamation-circle"></clr-icon>
-          </div>
-          <span class="alert-text">Your identity cannot be validated.</span>
+    <div class="card">
+      <div style="height: 5rem;">
+        <ngx-charts-pie-chart
+            [view]="view"
+            [scheme]="colorScheme"
+            [results]="spinnerValues"
+            [doughnut]="true"
+            [tooltipDisabled]="true">
+        </ngx-charts-pie-chart>
+        <div class="percentText">
+          {{completedTasksAsPercentage}}%
         </div>
+        <h6 style="width: 1rem; vertical-align: middle; width: 10rem; padding: 1rem; height: 100%">
+          Profile Completion Progress
+        </h6>
+      </div>
+      <div>
+        <button class="btn btn-secondary" routerLink="id-verification"
+            [disabled]="profile.blockscoreIdVerificationStatus === 'VERIFIED'">
+          Submit Identity Verification
+          <clr-tooltip *ngIf="profile.blockscoreIdVerificationStatus === 'REJECTED'">
+            <clr-icon clrTooltipTrigger shape="warning-standard"></clr-icon>
+            <clr-tooltip-content clrPosition="top-left" clrSize="md" *clrIfOpen>
+              <span>Your identity could not be verified. Please try again or contact an administrator</span>
+            </clr-tooltip-content>
+          </clr-tooltip>
+        </button>
+      </div>
+      <div>
+        <button class="btn btn-secondary" (click)="submitTermsOfService()"
+            [disabled]="email===null || this.profile.termsOfServiceCompletionTime !== null">
+          Submit Terms of Service
+        </button>
+      </div>
+      <div>
+        <button class="btn btn-secondary" (click)="completeEthicsTraining()"
+            [disabled]="email===null || this.profile.ethicsTrainingCompletionTime !== null">
```

Comment:
NIT: use spaces around operators (`===`) please.  See also lines 109 and 121 (at least).